eduNEXT/platform-global-teacher-campus
# feat: added names on endpoints

This PR adds the names required in the process validation endpoints.

How to Test

call the endpoint for exmaple: 
local.overhang.io:8000/plugin-cvw/api/v1/validation-processes/course-v1:test+CS101+2023_T1

As a response you should see the names as follows:

{
    "id": 2,
    "course": {
        "id": "course-v1:test+CS101+2023_T1",
        "display_name": "test"
    },
    "categories": [
        {
            "id": 1,
            "name": "courseCategory1",
            "parent_category": null
        }
    ],
    "organization": {
        "id": 2,
        "name": "test",
        "short_name": "test"
    },
    "current_validation_user": null,
    "validation_body": {
        "id": 1,
        "validators": [
            {
                "email": "admin@example.com"
            }
        ],
        "organizations": [
            {
                "id": 1,
                "name": "edX",
                "short_name": "edX"
            }
        ],
        "name": "validationBody1"
    },
    "events": [
        {
            "id": 2,
            "username": "admin",
            "create_at": "2023-09-15T17:56:17.948529Z",
            "status": "subm",
            "comment": "test comment",
            "validation_process": 2,
            "user": 4,
            "reason": null
        }
    ],
    "username": admin
}
